src/main/webapp/resources/js/contexts/AnalysisOutputsContext.js

```diff
@@ -71,7 +78,8 @@ function AnalysisOutputsProvider(props) {
               hasJsonFile: hasJsonFile,
               hasTabularFile: hasTabularFile,
               hasTextFile: hasTextFile,
-              hasExcelFile: hasExcelFile
+              hasExcelFile: hasExcelFile,
```

Comment:
Remember, in es6 this is redundant, you only need:

```js
fileTypes: [
            {
              hasJsonFile,
              hasTabularFile,
              hasTextFile,
              hasExcelFile,
              hasImageFile,
            }
          ]
```

Reply:
Updated in a6f973b9319bc3f2d954a416af2062bb5b776ba8